dashboard/src/components/SideMenu/SideMenu.tsx

```diff
@@ -49,6 +54,7 @@ const linkItems: LinkMenuItems[] = [
 const emptyFunc = (): void => {};
 
 const SideMenu = (): JSX.Element => {
+  const [selectedItemId, setSelectedItemId] = useState<number>(0);
```

Comment:
we can extract the selected item just by getting the path from the router and seeing if equals the path that the link brings us

Reply:
no need for useState here